Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs

```diff
@@ -14,16 +14,91 @@ static void Main(string[] args)
             Console.ReadKey(true);
         }
 
-        private static void Test(string args, int expected)
-        {
+        private static void Test(string args, int expected)
+        {
             var result = FindDigit(args).Equals(expected) ? "PASS" : "FAIL";
-            Console.WriteLine($"{args} : {result}");
+            Console.WriteLine($"{args} : {result}");
         }
-
-        public static int FindDigit(string equation)
-        {
+	// To find and replace the digit and then to check whether it is valid or not
+	public static int findMissing(int number,int position, String[] nums)
+        {
+            if(number.ToString().Length!= nums[position].Length)
+					{
+						return -1;
+					}
+					else
+					{
+						int find = nums[position].IndexOf('?');
+						string oldA = number.ToString();
+						char result = oldA[find];
+						String newA = nums[position].Replace('?',result);
+						if(string.Equals(newA,oldA))
+						{
+							return int.Parse(result.ToString());
+						}
+                   	    else
+						return -1;
+					}
+        }
+        
+        public static int FindDigit(string equation)
+        {
             // Add your code here.
-            throw new NotImplementedException();
+            char[]  splitchars = new char[2]{'*','='};
+            String[] nums=equation.Split(splitchars);
+            int check=-1;
+            for(int i=0;i<=2;i++)  
+            {
+                if(nums[i].Contains("?"))
+                {
+                    check=i;
+					break;

```

Comment:
If would also be useful to see if we can remove the break. Linear implementations without too many branches and breaks make for easier reading and maintainability.